src/data/builder-tools.js

```diff
@@ -769,6 +769,14 @@ export const Showcases = [
     website: "https://cexplorer.io",
     getstarted: null,
```

Comment:
If you hope to include this tool under the `Operator Tool` tag then it's suggested (and sometimes required) that you write a `getstarted:` page on the Developer Portal about it.  See other operator ools with Get Started pages beginning here: https://developers.cardano.org/docs/operate-a-stake-pool/guild-ops-suite

Reply:
My idea was more to add to the bottom of this page: https://developers.cardano.org/docs/operate-a-stake-pool/

Reply:
I could tell that from #805 but if everything of potential interest to operators was added at the bottom of this page then it would defeat the purpose of the `operatortool` tag in the Builder Tool section... as well as make that page too long, too difficult to classify, and/or too competitive to be included there (remember the IOG _Essential Cardano_ list?